src/core/ucc_context.c

```diff
@@ -18,6 +18,10 @@ static ucc_config_field_t ucc_context_config_table[] = {
      "An optimization hint of how many endpoints will be created on this context",
      ucc_offsetof(ucc_context_config_t, estimated_num_eps), UCC_CONFIG_TYPE_UINT},
 
+    {"MT_LOCK_FREE_PROGRESS_Q", "1",
```

Comment:
Naming: do we need MT in the name?

Reply:
lock free is only in MT.

Reply:
Yes, that is why it is redundant. It is implied. :)

Reply:
fixed